reposcan/repodata/repository_controller.py

```diff
@@ -76,24 +78,32 @@ def _download_repomds(self):
     def _check_cert_expiration_date(self, cert_name, cert):
         try:
             loaded_cert = crypto.load_certificate(crypto.FILETYPE_PEM, cert)
-            expire_date = datetime.strptime(loaded_cert.get_notAfter(), "%Y%m%d%H%M%SZ")
-            expire_in = expire_date - datetime.now()
-            if expire_in.days > 7:
-                self.logger.debug('Certificate %s is ok', cert_name)
-                CERT_EXPIRATION.labels(cert_name).state('valid')
-            elif 7 >= expire_in.days > 0:
-                self.logger.warning('Certificate %s will expire in %s', cert_name, expire_in.days)
-                CERT_EXPIRATION.labels(cert_name).state('expire_soon')
+            valid_to_dt = datetime.strptime(loaded_cert.get_notAfter(), "%Y%m%d%H%M%SZ")
+            expire_in_days_td = (valid_to_dt - datetime.utcnow()).days
+            expire_tuple = (valid_to_dt, expire_in_days_td)
+            if 7 >= expire_in_days_td > 0:
```

Comment:
Let's send the warning earlier, let's say 30 days. 7 days may not be enough time to issue new cert.